packages/sw-routing/src/lib/express-route.js

```diff
@@ -40,7 +40,6 @@ class ExpressRoute extends Route {
    */
   constructor({path, handler, method}) {
     assert.isType({path}, 'string');
-    assert.hasMethod({handler}, 'handle');
```

Comment:
In the course of considering whether I should write a test for it, I realized that the same assertion was being made in the constructor of the class it inherits from.

So it's not related to testing, but just something I snuck in.